x-pack/elastic-agent/docs/inputs/input-logfile.asciidoc

```diff
@@ -0,0 +1,163 @@
+:type: logfile
+
+[id="{beatname_lc}-input-{type}"]
+=== Logfile input
+
+++++
+<titleabbrev>Logfile</titleabbrev>
+++++
+
+Use the `logfile` input to read lines from log files.
+
+To configure this input, specify a list of glob-based <<input-paths,`paths`>>
+that must be crawled to locate and fetch the log lines.
+
+Example configuration:
+
+["source","yaml",subs="attributes"]
+----
+inputs:
+- type: logfile
+  streams:
+    - paths:
+      - /var/log/message
+      - /var/log/*.log
+----
+
+
+You can apply additional
+<<{beatname_lc}-input-{type}-options,configuration settings>> (such as `fields`,
+`include_lines`, `exclude_lines`, `multiline`, and so on) to the lines harvested
+from these files. The options that you specify are applied to all the files
+harvested by this input.
+
+To apply different configuration settings to different files, you need to define
+multiple stream or input sections:
+
+["source","yaml",subs="attributes"]
+----
+inputs:
+  - type: logfile
+    streams:
+     - paths:
+       - /var/log/system.log
+       - /var/log/wifi.log
+     - paths: "/var/log/apache2/*"
+       fields:
```

Comment:
Didn't we deprecated fields and `add_fields` should be used instead? @urso

Reply:
I believe we now recommend `add_fields` processor.